File: FCUBEAPI/FleetTrans/Models/TyreMgntReportList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FleetTrans.Models
{
    public class TyreMgntReportList
    {
        public List<TyreMgntReportModel> TyreList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }
}

File: OMKARAUTOAPP/src/app/models/tyremgntreportlist.ts
import { Paginationmodel } from "./paginationmodel";
import { Tyremgntreportmodel } from "./tyremgntreportmodel";

export class Tyremgntreportlist {
    tyreList: Tyremgntreportmodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}